docs/v-secure/guides/mcp-integration.md

```
@@ -0,0 +1,438 @@
+      // Only for specific operations
+      operations: ['read']
+    }
+  }
+});
+```
+
+## Use Cases
+
+### 1. Debugging Production Issues
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)